qiskit_experiments/library/randomized_benchmarking/layer_fidelity.py

```diff
@@ -152,6 +154,18 @@ def __init__(
             one_qubit_basis_gates: Optional, 1q-gates to use for implementing 1q-Clifford operations.
                             If not specified (but ``backend`` is supplied),
                             all 1q-gates supported in the backend are automatically set.
+            layer_barrier (bool): Optional, enforce a barrier across the whole layer.
```

Comment:
Can you add a release note about the new features?

Reply:
Added